addons/crm/models/res_partner.py

```diff
@@ -62,18 +62,19 @@ def _compute_meeting_count(self):
     def _compute_meeting(self):
         if self.ids:
             all_partners = self.with_context(active_test=False).search([('id', 'child_of', self.ids)])
+
+            event_id = self.env['calendar.event']._search([])  # ir.rules will be applied
```

Comment:
Could be great to include an even that the test user cannot read, just to be sures that ir rules are effectively applied.